manifests/local-storage.yaml

```diff
@@ -82,6 +82,26 @@ spec:
           valueFrom:
             fieldRef:
               fieldPath: metadata.namespace
+        livenessProbe:
+          exec:
+            command:
+              - /bin/sh
+              - -ec
+              - |
+                pgrep local-path-provisioner
+        readinessProbe:
+          exec:
+            command:
+              - /bin/sh
+              - -ec
+              - |
+                pgrep local-path-provisioner
+        resources:
+          requests:
+            cpu: 2m
```

Comment:
This is such a minor allocation we might as well just leave it unset, and only specify a memory request.

Reply:
So most of the contributions from today I got from inspecting my cluster with polaris (but kube-bench also works), and I'm basically trying to get a score as high as possible to reduce output during CI pipelines. Adding probes, resources and securityContext is part of that. Also, for doing stuff like Pod Security Context and Admission Controllers, I'd like to specify that all resource requests have to be filled out before a resource is allowed to be created.

If you think it's not meaningful, I'll remove it. But honestly I'd rather specify it than add exceptions to these kind of tools. I've removed the resources from the servicelb pod because you said it sleeps when done, so I agree it's not useful. local-path-provisioner keeps running.